examples/tutorials/setup_poc.ipynb

```diff
@@ -429,15 +462,75 @@
     "\n",
     "When starting the POC deployment, it's necessary to use a separate terminal since the `nvflare poc start` command will run  in the foreground emitting output from the server and any connected clients.\n",
     "\n",
-    "Also note that `nvflare poc start` starts all participants, including the admin console. We want to start the 2 clients and 1 server, but without the FLARE console (aka Admin Console), as we are going to interact with the system from a notebook.\n",
+    "Also note that `nvflare poc start` starts all clients and server as well as project admin console. All other participants (org admin, lead and member) will not started. \n",
+    "\n",
+    "In many cases, we want to start the clients and server, but without the FLARE console for project admin, as we are going to interact with the system from a notebook or CLI. You first need to find out the user name for the project admin. By default, this is admin@nvidia.com\n",
     "\n",
     "Create a terminal to execute the following command (in JupyterLab or host terminal), then\n",
     "\n",
     "```\n",
     "    nvflare poc start -ex admin@nvidia.com\n",
     "```\n",
-    "**Note:**\n",
-    "    Using ```%%bash -bg``` to run the above command in a code cell may not always work"
+    ">**Note:**\n",
+    "    Using ```%%bash -bg``` to run the above command in a code cell may not always work\n",
+    "    \n",
+    "#### A more complex example\n",
+    "    \n",
+    "In this example, lets see how to start the individual user participants defined in custom_project.yml. Note the project admin is \"admin@nonprofit.org\" instead of \"admin@nvidia.com\"\n",
+    "\n",
+    "Please use terminal to try these commands\n",
+    "\n",
+    "* We start start without project admin console\n",
+    "\n",
+    "```\n",
+    "    nvflare poc start -ex admin@nonprofit.org\n",
+    "```\n",
+    "\n",
+    "This will start two hospital clients and server. \n",
+    "\n",
+    "* Next we like to start US hospital's lead's FLARE console \n",
+    "\n",
+    "```\n",
+    "\n",
+    "nvflare poc start -p lead@hospital.org.us\n",
+    "\n",
+    "```\n",
+    "\n",
+    "You are acting as US hospital's lead \n",
+    "\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da2fd969-1886-4964-b122-129fab2113df",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "! nvflare poc stop"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f4f80e58-92d2-4dca-be1a-a909ede39fca",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
```

Comment:
For unknown reasons, this notebook file can not be rendered on GitHub website, maybe this part needs to be removed?

Reply:
I don't have the issue for GitHub rendering this file. Maybe it's just a one time GitHub issue you experienced.

Reply:
main branch rendered. this file does not.
I am using Firefox with Ubuntu OS

Reply:
I don't have issue for reentering the setup_poc.ipynb file, Firefox + Ubuntu
also works on Mac